sycamore/transforms/partition.py

```diff
@@ -46,10 +46,17 @@ def element_in_left_col(e: Element) -> bool:
 class Partitioner(ABC):
     @staticmethod
     def to_element(dict: dict[str, Any]) -> Element:
+        text = dict.pop("text")
+        if isinstance(text, str):
+            binary = text.encode("utf-8")
+        else:
+            binary = text
+            text = str(binary, "utf-8")
+
         element = Element()
         element.type = dict.pop("type")
-        element.binary_representation = dict.pop("text")
-        element.text_representation = str(element.binary_representation)
+        element.binary_representation = binary
+        element.text_representation = text
```

Comment:
Hmm. It actually looks like the type of `element.binary_representation` is wrong. It is `Optional[str]`, but I think it should be `Optional[bytes]`. `Document` has it right, so I'm guessing this is just an oversight. @bohou-aryn just confirming there's not a reason you know of? I noticed b/c this is failing type checking since you have correctly assigned bytes.

Reply:
(to be clear, the issue I'm referring to is not in the code here, but in the definition of the Element class).

Reply:
Makes total sense.  That's what I get for only running 2 of the 3 linters locally.  We should get Poetry to run all the linters in one command.

Reply:
`poetry run pre-commit run --all-files` should run all the linters, though there are some issues running mypy in pre-commit that I haven't been able to figure out (https://github.com/aryn-ai/sycamore/issues/83). In the meantime `poetry run mypy --install-types .` should run mypy in the most reliable way.

Reply:
I don't know about that type change, mostly the reason should just be a typo when restructuring. Didn't we decide to only keep the bytes representation? this is postponed to future fix?